packages/react/src/components/FileSelect/FileSelect.tsx

```diff
@@ -40,14 +41,24 @@ export const DEFAULT_PROPS = {
  * @internal @unstable
  */
 export const FileSelect = React.forwardRef<HTMLInputElement, FileSelectProps>(
-  function FileSelect({ multiple = true, type = 'file', ...props }, ref) {
+  function FileSelect(
+    {
+      multiple = true,
+      testId = 'amplify-file-select',
+      type = 'file',
+      ...props
+    },
+    ref
+  ) {
     return (
       <input
         {...DEFAULT_PROPS}
-        {...(type === 'folder' ? { webkitdirectory: '' } : undefined)}
         {...props}
+        {...(type === 'folder' ? { webkitdirectory: '' } : undefined)}
+        data-testid={testId}
         multiple={multiple}
         ref={ref}
+        type="file"
```

Comment:
Shouldn't this be `type={type}` ? We have it as a prop, but then we are throwing it away?

Oh I see now, the `type` prop is `'file' | 'folder'` and NOT the HTML type attribute for an input. Could we put a comment either here or where we are destructuring `type` so future people don't get confused?